src/components/Shared/Avatar.vue

```diff
@@ -7,10 +7,9 @@
       {{ initials || '' }}
```

Comment:
It appears that after removing `'??'` from the `initials` assignment, we should use this value instead of `''` to ensure something is displayed under the avatar, maintaining the layout integrity.